components/Conversation.tsx

```diff
@@ -41,24 +42,23 @@ export default function Conversation({
 
   const [newMessage, setNewMessage] = useState('')
   const [messagesByDay, setMessagesByDay] = useState<MessagesOfADay[]>([])
-  const [uploadedFileInfo, setUploadedFileInfo] = useState<InfoFichier | null>(
-    null
-  )
+  const [uploadedFileInfo, setUploadedFileInfo] = useState<
+    InfoFichier | undefined
+  >(undefined)
+  const [uploadedFileError, setUploadedFileError] = useState<
+    string | undefined
+  >(undefined)
+  const [fileUploading, setFileUploading] = useState<boolean>(false)
```

Comment:
```suggestion
  const [fileUploading, setFileUploading] = useState<boolean>(false)
```
on préfixe pas avec is? (pour se mettre d'accord si Boolean tout ça :-) )